controllers/src/main/scala/cromwell/pipeline/controller/ProjectController.scala

```diff
@@ -24,6 +24,17 @@ class ProjectController(projectService: ProjectService) {
     }
   }
 
+  private def getProjects(implicit accessToken: AccessTokenContent): Route = get {
+    onComplete(projectService.getUserProjects(accessToken.userId)) {
+      case Success(projects) =>
+        projects match {
+          case Nil => complete(StatusCodes.NotFound)
```

Comment:
Shouldn't we treat an empty seq as completely valid result here (without any errors)

Reply:
What do you mean?

Reply:
That StatusCode NotFound tells us that the entity cannot be found. But in that case an empty entity is a valid entity, that was found. But I can be hugely wrong about that )

Reply:
We can use different approach, I selected this :)

Reply:
I agree with @Nikolai-Isaev. Empty list looks valid to me in this case

Reply:
Ok. I fixed it.